Makefile

```diff
@@ -1,12 +1,9 @@
 DOCKER_REVISION ?= testing-$(USER)
 DOCKER_TAG = docker-push.ocf.berkeley.edu/jukebox:$(DOCKER_REVISION)
-RANDOM_PORT := $(shell expr $$(( 8000 + `id -u` % 1000 )))
 
-
-.PHONY: dev
-dev: cook-image
-	@echo "Will be accessible at http://$(shell hostname -f ):$(RANDOM_PORT)/"
-	docker run --rm -p "$(RANDOM_PORT):8000" "$(DOCKER_TAG)"
+.PHONY: test
```

Comment:
Very small nitpick, but I wonder if `make test` should be reserved for running unit tests. I think the old `make dev` made more sense as a name.

Reply:
I think @encadyma misread this